### DISCUSSION TITLE
help with postal please

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2337] [Help with using Postal]
i have two servers from the same host provider, installed postal on both with the same configuration and all. Upn sending message to outlook.com, it is sent on one and rejected in the other, how do i get to fix this please?
![failed](https://user-images.githubusercontent.com/96216385/214280297-aaa47d29-fa54-48e8-9c9d-16df38fce06d.png)
![sent](https://user-images.githubusercontent.com/96216385/214280320-eac27543-60ff-4cfc-b3e5-98012f2739bf.png)

**Comment by willpower232** [comment]
Looks like you need to check both servers with mxtoolbox.com to see if you're on a block list, as per the error message
